composer.json

```diff
@@ -28,14 +20,14 @@
   ],
   "repositories": [{
     "type": "vcs",
-    "url": "https://github.com/xibosignage/support"
+    "url": "https://github.com/dasgarner/support"
```

Comment:
After we merge the other PR, I guess this could still point to `xibosignage` support repo? 🤔 
Unless we want to keep like that for now?

Reply:
You're right - i'll update!